tests/lib/snaps/test-snapd-policy-app-consumer/meta/snap.yaml

```diff
@@ -287,54 +188,24 @@ apps:
   mount-control:
     command: bin/run
     plugs: [ mount-control ]
-  mount-observe:
-    command: bin/run
-    plugs: [ mount-observe ]
   mpris:
     command: bin/run
     plugs: [ mpris ]
   multipass-support:
     command: bin/run
     plugs: [ multipass-support ]
-  netlink-audit:
-    command: bin/run
-    plugs: [ netlink-audit ]
-  netlink-connector:
-    command: bin/run
-    plugs: [ netlink-connector ]
-  network:
-    command: bin/run
-    plugs: [ network ]
-  network-bind:
-    command: bin/run
-    plugs: [ network-bind ]
-  network-control:
-    command: bin/run
-    plugs: [ network-control ]
   network-manager:
     command: bin/run
     plugs: [ network-manager ]
```

Comment:
There's also a test for this although it doesn't explicitly have a `snap connect` since it connects by default. Maybe we can either relax the check or just add the connect at the end to cut this one out as well